source/common/http/conn_manager_impl.cc

```diff
@@ -1531,18 +1531,22 @@ void ConnectionManagerImpl::ActiveStream::onBelowWriteBufferLowWatermark() {
 }
 
 Tracing::OperationName ConnectionManagerImpl::ActiveStream::operationName() const {
+  ASSERT((connection_manager_.config_.tracingConfig()));
```

Comment:
Here and in other places, could you add comments re: expected usage of this api please?